### PR TITLE
Feature/create search panel

### PR DESCRIPTION
**Pull request by rshulyar**

### CODE REVIEW

**Comment by rshulyar** [comment]
https://rshulyar.atlassian.net/browse/GH-14

File: src/components/logo/logo.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import logo from './../../img/logo.png';
+import styles from './logo.module.css';
+
+function Logo() {
+    return(
+        <div className={styles["wrap"]}>
+            <div className={styles["inner"]}>
+                <img className={styles["image"]} src={logo}></img>
```
Comment (dobryanskyy) [inline src/components/logo/logo.js]: Please use self-closing tags if there is no children

File: src/components/logo/logo.module.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    color: rgb(34, 34, 34);
+    display: flex;
+    flex-direction: column;
+    flex-shrink: 0;
+    font-family: arial, sans-serif;
+    font-size: 14px;
+    height: 92px;
+    max-height: 290px;
+    min-height: 92px;
+    /* width: 1440px; */
```
Comment (dobryanskyy) [inline src/components/logo/logo.module.css]: ?

File: src/components/logo/logo.module.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    position: relative;
+    width: 272px;
+    /* margin-top: 68px;  */
+}
+
+/* @media screen and (min-height: 666) {
+    .inner {
+        margin-top: 0px;
+    }
+} */
```
Comment (dobryanskyy) [inline src/components/logo/logo.module.css]: !

File: src/components/searchPanel/search/search.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+function Search() {
+
+    return (
+        <div className={styles["wrap"]}>
+            <img src={dandruffImage} className={styles["dandruff-image"]}></img>
+            <input className={styles["search-input"]}></input>
+            <div className={styles["keyboard-and-microphone-images"]}>
+                <img src={keyboardImage} className={styles["keyboard-image"]}></img>
+                <img src={microphoneImage} className={styles["microphone-image"]}></img>
```
Comment (dobryanskyy) [inline src/components/searchPanel/search/search.js]: self closing

File: src/components/searchPanel/search/search.module.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    display: flex;
+    padding-right: 14px;
+    padding-top: 5px;
+    padding-left: 8px;
+    background-color: white;
+    margin-left: 7px;
+}
+
+.wrap:hover {
+    /*  */
```
Comment (dobryanskyy) [inline src/components/searchPanel/search/search.module.css]: ?

**Pull request merged**